client/components/ReviewRating/ReviewList/ReviewList.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Review from './Review';
```

Comment:
Can we add a line of spacing here between package and component imports?

Reply:
I will be mindful of the future code.